app/views/support_interface/application_forms/delete_application/confirm_delete.html.erb

```diff
@@ -29,7 +29,10 @@
         <%= f.govuk_check_box :accept_guidance, true, multiple: false, label: { text: 'I have read the guidance' }, link_errors: true %>
       <% end %>
 
-      <%= f.govuk_submit %>
+      <% if HostingEnvironment.production? %>
+        <%= govuk_warning_text(text: 'You are about to delete an account in the PRODUCTION environment.') %>
+      <% end %>
+      <%= f.govuk_submit('Delete candidate account', warning: true) %>
```

Comment:
```suggestion
      <%= f.govuk_submit('Delete candidate account in production', warning: true) %>
```

Reply:
Not a bad suggestion but we'll need to conditionally change the button if we do that

Reply:
I don't feel super strongly about it. What you have there should be enough. I just thought its a bit more in your face, which might be good.